packages/math-input/src/types.ts

```diff
@@ -105,3 +106,16 @@ export interface KeypadAPI {
     setKeyHandler: (keyHandler: KeyHandler) => void;
     getDOMNode: () => ReturnType<typeof ReactDOM.findDOMNode>;
 }
+
+export type KeypadContext = {
```

Comment:
**suggestion**: I'm mildly concerned about having a `keypadContext` and `KeypadContext`. 🤔  I also think that typically our contexts are named `TitleCase`.  So, what do you think about naming this type `KeypadContextType` and then keeping the naming of the original `KeypadContext` as it was.  This `KeypadContextType` looks like it's not needed outside of the `math-input` package so a slightly clumsy name internally feels easier to deal with.